test/.eslintrc.yml

```diff
@@ -0,0 +1,13 @@
+env:
+  jest: true
+globals:
+  afterAll: true
```

Comment:
doesn't adding `jest: true` under the `env` key include all the testing globals such as `afterEach`?

Reply:
I’ll do some more investigation here. I believe they were required — but I’ll see if I can remove it. If they’re required, I’ll figure out what `env: jest: true` does… perhaps it’s meaningless.

Reply:
Good call! They are not required — removed in 03e2aa0.